docs/github-actions/learning-resources.md

```diff
@@ -1,8 +1,9 @@
 ---
-sidebar_position: 3
+sidebar_position: 4
```

Comment:
It's a side position for the resources page on the website not the number of resources we have on the page.
 
```suggestion
sidebar_position: 3
```